tests/issues/test_move_issues.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from pds_github_util.issues.move_issues import move_issues, get_gh_connection
+
+
+class MyTestCase(unittest.TestCase):
+
+    def test_move_issue(self):
+        gh_connection = get_gh_connection()
+        move_issues('NASA-PDS/pds-api-javalib', 'NASA-PDS/registry-api', gh_connection, label='model', dry_run=False)
```

Comment:
I see `dry_run=False` here, so does that mean every time someone runs the test case issues will get moved from `pds-api-javalib` → `registry-api`?

Reply:
good catch @nutjob4life , I will switch to dry_run=True to avoid calling github api, since I don't mock it.